src/Microsoft.AspNet.Mvc.Core/Logging/ControllerModelValues.cs

```diff
@@ -83,6 +84,11 @@ public ControllerModelValues(ControllerModel inner)
         /// </summary>
         public List<AttributeRouteModelValues> AttributeRoutes { get; set; }
 
+        /// <summary>
+        /// Gets the arbitrary metadata properties associated with the controller <see cref="ControllerModel.Properties"/>.
```

Comment:
note that the comments are not consistent.

Please follow this up in a PR where you add the full XML docs for the files in this PR